firefox/janitor-hg.json

```diff
@@ -30,6 +34,6 @@
     },
     "Clobber build": "./mach clobber",
     "Update source code": "hg pull -u",
-    "Send to code review": "hg push review"
+    "Send to code review": "arc diff"
```

Comment:
Can you leave `hg push review` for now until `arc diff` is stable enough ? Or provide both options ?

Reply:
Sure! But `arc diff` seemed pretty stable to me when I tried it like a month ago. What are your gripes with it?

Reply:
- Multi-part bugs are really hard to push. 
- There's no autoland yet
- `r=...` works unreliably in commit messages

Reply:
> Multi-part bugs are really hard to push.

Well ok, I'll keep both options available in the `hg` image until MozReview stops existing.

The `git` image only supported raw patch uploads, and that's been broken for a while I think, so Phabricator will be an improvement.

> There's no autoland yet

There is! Since last week. 😏 

> `r=...` works unreliably in commit messages

Next time it breaks, please file a bug here: https://bugzilla.mozilla.org/enter_bug.cgi?product=Conduit&component=Phabricator